api/copy_chunk_experimental.md

```diff
@@ -0,0 +1,27 @@
+## copy_chunk() <tag type="community">Community</tag>
+TimescaleDB allows you to copy existing chunks to a new location within a
+multi-node environment. This allows each data node to work both as a primary for
+some chunks and backup for others. If a data node fails, its chunks already
+exist on other nodes that can take over the responsibility of serving them.
+
+### Required arguments
+
+|Name|Type|Description|
+|-|-|-|
```

Comment:
Argument data types
```
chunk regclass, 
source_node name, 
destination_node name
```

Reply:
There's no support for tablespaces YET while copying chunks across the data nodes.